app/src/main/res/layout/fragment_add_loan_application.xml

```diff
@@ -201,17 +199,17 @@
                     android:layout_weight="0.3"
                     android:layout_width="wrap_content">
 
+                    <ImageView
+                        android:layout_height="match_parent"
+                        android:layout_width="30dp"
+                        app:srcCompat="@drawable/ic_edit_black_24dp"/>
+
```

Comment:
@ritish099 edit button that is placed to left of Expected date doesn't look good. It would be better to keep previous layout or instead remove edit button and make `tv_expected_disbursement_date` clickable to do work of edit btn. What do you say?

Reply:
@PatelVatsalB21 old layout was not aligned. The alignment of currency data field was not matching with the other two fields.
and
if we remove the edit button altogether and make "tv_expected_disbursement_date" clickable. then it can be hard for the user to understand that it is editable.

Reply:
@ritish099 i agree with your point but this type of edit btn is very weird looking as we all you apps, but i haven't such alignment of button. Other way is to bring currency to translate currency to little left instead of these btn and 2 tvs.